conwayste/src/ui/layer.rs

```diff
@@ -696,4 +718,34 @@ mod test {
         let removal = layer_info.remove_widget(pane_id.clone());
         assert_eq!(removal.is_ok(), false);
     }
+
+    #[test]
+    fn test_remove_widget_adds_id_to_hashset() {
+        let mut layer_info = Layering::new();
+
+        let pane = Pane::new(Rect::new(0.0, 0.0, 1.0, 1.0));
+        let pane_id = layer_info
+            .add_widget(Box::new(pane), InsertLocation::AtCurrentLayer).unwrap();
+
+        let removal = layer_info.remove_widget(pane_id.clone());
+        assert_eq!(removal.is_ok(), true);
+
+        assert!(layer_info.removed_node_ids.contains(&pane_id));
+    }
+
+        #[test]
+        #[should_panic]
```

Comment:
Please de-indent these two lines. Also, what is panicking here?

If it's an unwrap, I'd rather have `assert_eq!` to check that it's a certain error value.

Reply:
Whoops, thanks. I can see the confusion since there are multiple calls to `unwrap()`.